rusk-uplink/src/helpers.rs

```diff
@@ -0,0 +1,19 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use core::hash::Hash;
+use std::collections::hash_map::DefaultHasher;
```

Comment:
This is yet another conditional compilation candidate. We're including `std` here.